pyspark_ai/prompt.py

```diff
@@ -51,28 +64,32 @@
 Write a Spark SQL query to retrieve from view `spark_ai_temp_view_93bcf0`: Pivot the fruit table by country and sum the amount for each fruit and country combination.
 Thought: Spark SQL does not support dynamic pivot operations, which are required to transpose the table as requested. I should get all the distinct values of column country.
 Action: query_sql_db
-Action Input: "SELECT DISTINCT Country FROM spark_ai_temp_view_93bcf0"
+Action Input: "SELECT DISTINCT `Country` FROM spark_ai_temp_view_93bcf0"
 Observation: USA, Canada, Mexico, China
 Thought: I can write a query to pivot the table by country and sum the amount for each fruit and country combination.
 Action: query_validation
-Action Input: SELECT * FROM spark_ai_temp_view_93bcf0 PIVOT (SUM(Amount) FOR Country IN ('USA', 'Canada', 'Mexico', 'China'))
+Action Input: SELECT * FROM spark_ai_temp_view_93bcf0 PIVOT (SUM(Amount) FOR `Country` IN ('USA', 'Canada', 'Mexico', 'China'))
 Observation: OK
 Thought:I now know the final answer.
-Final Answer: SELECT * FROM spark_ai_temp_view_93bcf0 PIVOT (SUM(Amount) FOR Country IN ('USA', 'Canada', 'Mexico', 'China'))"""
-    """QUESTION: Given a Spark temp view `spark_ai_temp_view_19acs2` with the following columns:
+Final Answer: SELECT * FROM spark_ai_temp_view_93bcf0 PIVOT (SUM(Amount) FOR `Country` IN ('USA', 'Canada', 'Mexico', 'China'))"""
+    """QUESTION: Given a Spark temp view `spark_ai_temp_view_12qcl3` with the following columns:
 ```
-Car STRING
-Color STRING
-Make STRING
+Student STRING
+Birthday STRING
 ```
-Write a Spark SQL query to retrieve from view `spark_ai_temp_view_19acs2`: If color is gold, what is the total number of cars?
-Thought: I will use the column 'Color' to filter the rows where its value is 'gold' and then select the COUNT(`Car`) 
-because COUNT gives me the total number of cars.
+Write a Spark SQL query to retrieve from view `spark_ai_temp_view_12qcl3`: What is the total number of students with the birthday January 1, 2006?
+Thought: I need to filter by an exact birthday value from the df. I will use the tool similar_value to help me choose my filter value.
+Action: similar_value
+Action Input: January 1, 2006|Birthday|spark_ai_temp_view_12qcl3
```

Comment:
In the followup PR for tests, we should test the similar scores of January 1, 2006 with some dates